PROD-2791: replace succeeded and failed with success and failure

nitishkumar71 opened the pull request:
We are standardizing the `event_action` values across different integrations.

nitishkumar71 committed:
replace succedded and failure with success and failure

Signed-off-by: Nitishkumar Singh <nitish@cto.ai>

marcoguillenuio commented:
@nitishkumar71 cc @jossemargt-cto-ai 
A general comment about the standardization. Great that we are standardizing, I have seen throughout the code some different event actions / combinations for the same action, it would be good to review the components where these variations are used.

E.g.:
<img width="1063" alt="feedbackreport" src="https://github.com/cto-ai/action/assets/101744963/a18c87d2-2e79-4f63-9482-07de1d40d7f4">

nitishkumar71 commented:
> @nitishkumar71 cc @jossemargt-cto-ai 
> 
> A general comment about the standardization. Great that we are standardizing, I have seen throughout the code some different event actions / combinations for the same action, it would be good to review the components where these variations are used.
> 
> 
> 
> E.g.:
> 
> <img width="1063" alt="feedbackreport" src="https://github.com/cto-ai/action/assets/101744963/a18c87d2-2e79-4f63-9482-07de1d40d7f4">
> 
> 

Let's keep the discussion at one place.

https://github.com/cto-ai/action/pull/17#issuecomment-1585506308

NickAnderegg committed:
docs: copyedit README.md

- Reformatted existing content to match the structure for our `events-pipe` readme.
- Fixed the ordering of the Team ID and API Token in the example

NickAnderegg reviewed (approved):
I've made copyediting changes of my own and committed to this branch. Overall, LGTM.
